test/src/tests.js

```diff
@@ -139,4 +174,577 @@ describe('Bing Ads Event Forwarder', function() {
             done();
         });
     });
+
+    describe('Consent', function() {
+        var consentMap = [
+            {
+                jsmap: null,
+                map: 'marketing_consent',
+                maptype: 'ConsentPurposes',
+                value: 'ad_storage',
+            },
+        ];
+
+        beforeEach(function() {
+            mParticle.forwarders = [];
+        });
+
+        afterEach(function() {
+            window.uetq = [];
+        });
+
+        it('should consolidate consent information', function(done) {
```

Comment:
what does it mean to consolidate consent information? can this test name be more specific?

Reply:
"Should parse the consent information from an mParticle event and log that" or something like that